src/part05.rs

```diff
@@ -47,7 +47,8 @@ impl BigInt {
         }
     }
 
-    // We can convert any vector of digits into a number, by removing trailing zeros. The `mut`
+    // Any vector of digits, which meets the structure of BigInt's `data` field, can be easily
```

Comment:
When I did Exercise 05.1 it was somehow natural for me to treat my Vec<u64> as big endian, e.g.  vec![1,2,0] for 0\*64^0 + 2\*64^1 + 1\*64^2. So my from_vec() implementation had to fill the BigInt data in reverse, whereas your implementation just copied the complete vector into the struct's data field. The examples and unit tests later in the course wouldn't fit with my implementation. No big deal, but maybe we could give a hint for future learners.